theano/tensor/tests/test_opt.py

```diff
@@ -1558,8 +1558,7 @@ def test_log_add():
         f([10000], [10000])  # causes overflow if handled incorrectly
         assert numpy.allclose(f([10000], [10000]), 20000)
     except AssertionError:
-        raise KnownFailureTest(('log(add(exp)) is not stabilized when adding '
-                'more than 2 elements, see #623'))
+        raise SkipTest("log(add(exp)) is not stabilized when adding more than 2 elements, see #623")
```

Comment:
The reason this error message was on two lines was to avoid lines longer than 79 characters, as per pep8.